docs/concepts/storage/persistent-volumes.md

```diff
@@ -695,6 +695,8 @@ parameters:
 
 #### Azure Disk
 
+##### Azure Unmanaged Disk Storage Class (before v1.7.2)
```

Comment:
@rootfs yes, it's still supported, and compatible after v1.7.2, I have updated the doc